src/inputs/actions.js

```diff
@@ -49,6 +65,7 @@ export function updateInputValue(input_id, value) {
         }))
         .then((inputs) => {
             dispatch(updateInputDefs(inputs));
+            dispatch(endUpdateInputValue());
```

Comment:
just thought it would be nice to be explicit here and have a matching action for beginUpdateInputValue